remote_config/src/android/remote_config_android.cc

```diff
@@ -496,8 +496,14 @@ static void JConfigInfoToConfigInfo(JNIEnv* env, jobject jinfo,
                                     ConfigInfo* info) {
   FIREBASE_DEV_ASSERT(env->IsInstanceOf(jinfo, config_info::GetClass()));
 
-  info->fetch_time = env->CallLongMethod(
+  int64_t fetch_time = env->CallLongMethod(
       jinfo, config_info::GetMethodId(config_info::kGetFetchTimeMillis));
+  // The C++ fetch_time is a uint64_t, so if we are given a negative number,
+  // use 0 instead, to prevent getting a very large number.
+  if (fetch_time < 0) {
+    fetch_time = 0;
+  }
+  info->fetch_time = fetch_time;
```

Comment:
Probably we should check for exceptions here as well, before the GetLastFetchStatus call.

Reply:
Yeah, good call out